README.md

```diff
@@ -584,7 +585,19 @@ After doing so, you can run the `*SC` TCP server commands, specifying the simple
 STRTCPSVR SERVER(*SC) INSTANCE('kafka')
 ```
 
-**Running two or more STRTCPSVR commands simultaneously**
+**Important Note: As of Service Commander v1.2.x, the TCP server is set to `AUTOSTART(*YES)` by default. If this
+is not the desired behavior, change it with the CHGTCPSVR command. For instance:**
+```bash
+CHGTCPSVR SVRSPCVAL(*SC) AUTOSTART(*YES)
```

Comment:
Shouldn't this be "AUTOSTART(*NO)"?

Reply:
This is why I'm glad to have someone reviewing!!

Reply:
👍